proposals/devcontainer-features.md

```diff
@@ -91,35 +52,36 @@ Options
 | id.default | string | Default value for the option. |
 | id.description | string | Description for the option. |
 
-## devcontainer-collection.json properties
-
-A feature collection file is a compilation of the `devcontainer-feature.json` files for each individual feature. It inlines all the information in each `devcontainer-feature.json` for each feature.
-
-If the application finds a `devcontainer-collection.json` file in the root of a downloaded tar file, then it uses that file to find the particular feature that will be executed.
+## devcontainer.json properties (feature related)
 
-In addition to the list of features included, the file includes the following properties.
+Features are referenced in `devcontainer.json` under the top level `features` object. 
 
+The properties are:
 | Property | Type | Description |
 | :--- | :--- | :--- |
-| name | string | Name of the collection. |
-| reference | string | Reference information of the repository and path where the code is stored. |
-| version | string | Version of the code. |
+| id | string | Reference to the particular feature to be included. |
+| options | object | Type of the option .|
```

Comment:
This seems a bit misleading to me. Suggestion: the `features` object contains keys, and each key is a feature reference (see format below), and each value is an object containing the options to be passed to the respective feature. 

Maybe an example would be useful here e.g.:
```
// ...
"features": {
  "ghcr.io/user/repo/go:1": {},
  "https://example.com/path.tgz": { "specificOption": "value" }
  "./relativeFolderPath": { "anotherOption": true }
}
// ...
```

Reply:
Good call out, this section was a carry over from the previous proposal but could definitely use more details.  Will edit!